UI/assets/css/allCandidate.css

```diff
@@ -0,0 +1,89 @@
+.candidate-card {
+	border: 1px solid rgba(0,0,0,.125);
+	border: 1px solid #81d4fa;
+	width: 100%;
+	height: 160px;
+	padding: 0 20px;
+	border-radius: 5px;
+	position: relative;
+	display: flex;
+	flex-direction: column;
+    min-width: 0;
+    word-wrap: break-word;
+    background-clip: border-box;
+    transition-duration: 1s;
+    margin: 20px 0;
+    background-color: #ffffff;
+}
+
+.candidate-card:first-child, .candidate-card:last-child {
+	margin: 0;
+}
+
+.candidate-card:hover {
+	box-shadow: 0 4px 8px 0 rgba(0,0,0,.2), 0 6px 20px 0 rgba(0,0,0,.19);
+}
+
+.candidate-card-title {
+	margin: 15px 0  6px;
+	color: #343a40;
+}
+
+.candidate-card-profile {
+	font-size: 15px;
+	color: rgba(0,0,0,.54);
+}
+
+.candidate-card-button-area {
+	 margin: 8px 0;
+}
+
+.candidate-card-button-area >a button {
+	cursor: pointer;
+	padding: 14px 50px;
+	background-color: #29b6f6;
+	border: none;
+	transition-duration: 0.4s;
+	border-radius: 5px;
+	color: #ffffff;
+	margin-right: 10px;
+
+}
+
+.candidate-card-footer {
+	text-align: right;
+    font-weight: 500;
+    /*margin-top: 47px;*/
+    color: rgba(0,0,0,.54);
```

Comment:
Kindly fix these inconsistent indentations, and also, delete commented code blocks.